projects/igniteui-angular/src/lib/grids/grid.common.ts

```diff
@@ -654,6 +654,12 @@ export class ContainerPositioningStrategy extends ConnectedPositioningStrategy {
             container.getBoundingClientRect().bottom <
                 target.getBoundingClientRect().bottom + contentElement.getBoundingClientRect().height;
 
+        // Change wrapper position from 'fixed' to 'absolute'
+        // so that it is hidden when scrolled below the parent grid body content.
+        contentElement.parentElement.style.position = 'absolute';
```

Comment:
Wait, shouldn't this be implemented in the component theme, rather than in here?

Reply:
That's the overlay wrapper - the one that's usually full viewport (fixed).
Just for this case we make it absolute so the wrapper can be affected by the grid tbody overflow setting.
This could be made into a modifier in the overlay theme to make it generally reusable in the future.

Reply:
You should add a class to it.

Reply:
I am all for making it a modifier class.